src/components/TenantSelector/stories/basic.story.mdx

```diff
@@ -0,0 +1,299 @@
+      title="Example app"
+      onTenantSelected={onTenantSelected}
+      validateTenant={validateTenantFailed}
+    />
+  </Story>
+</Preview>
+
+### Custom Error Message On Validation Error
+
+<Preview>
+  <Story name="Custom Error Message On Validation Error" >
+    <TenantSelector
+      title="Example app"
+      onTenantSelected={onTenantSelected}
+      unknownMessage="ARE YOU NUTS?!"
+      validateTenant={validateTenantFailed}
+    />
+  </Story>
+</Preview> 
+
+### Loads Forever
+
+```jsx
+const loadsForEver = () => Promise.race([]);
+```
+
+<Preview>
+  <Story name="Loads Forever" >
+    <TenantSelector
+      title="Example app"
+      onTenantSelected={onTenantSelected}
+      validateTenant={validateTenantForever}
+    />
+  </Story>
+</Preview>
+
+### Initial Tenant
+
+<Preview>
+  <Story name="Initial Tenant" >
+    <TenantSelector
+      title="Example app"
+      onTenantSelected={onTenantSelected}
+      initialTenant="sample-tenant"
+    />
+  </Story>
+</Preview>
+
+### Placeholder Tenant
+
+<Preview>
+  <Story name="Placeholder Tenant" >
+    <TenantSelector
+      title="Example app"
+      onTenantSelected={onTenantSelected}
+      placeholder="Enter your company's CDP project name"
+    />
+  </Story>
+</Preview>
+
+### Advanced Options
+
+```jsx
+const advancedOptions = { apiUrl: '', comment: 'Comment' };
+```
+
+<Preview>
+  <Story name="Advanced Options" >
+    <TenantSelector
+      title="Example app"
+      onTenantSelected={onTenantSelected}
+      advancedOptions={advancedOptionsProps}
+    />
+  </Story>
+</Preview>
+
+### With Custom Styles
+
+```jsx
+const customStyle = {
+  title: {
+    color: 'red',
+    alignSelf: 'center',
+    fontFamily: 'Comic Sans MS',
+  },
+  subTitle: {
+    color: 'purple',
+    alignSelf: 'center',
+  },
+  wrapper: {
+    width: 400,
+    backgroundColor: '#ffffa7',
+    borderRadius: 30,
+    boxShadow: 'none',
+  },
+  button: {
+    width: 200,
+    textTransform: 'none',
+    alignSelf: 'center',
+    borderRadius: 10,
+    backgroundColor: 'magenta',
+    color: 'white',
+  },
+  input: {
+    borderRadius: 10,
+    border: '2px solid #33DD33',
+  },
+  collapseWrapper: {
+    backgroundColor: '#ffffa7',
+  },
+};
+```
+
+<Preview>
+  <Story name="With Custom Styles" >
+    <TenantSelector
+      title="Example app"
+      onTenantSelected={onTenantSelected}
+      advancedOptions={advancedOptionsProps}
+      styles={customStyle}
+    />
+  </Story>
+</Preview>
+
+### With Theme
+
+```jsx
+import { ThemeProvider } from 'styled-components';
+
+const themeExample = {
+  gearbox: {
+    primaryColor: 'orange',
+    textColor: '#999',
+    containerColor: '#F4F4F4',
+    lightGrey: 'white',
+    buttonDisabledColor: '#DDD',
+    lightShadow: 'rgba(0, 0, 0, 0.15) 10px 10px 8px 8px',
+  },
+};
+```
+
+<Preview>
+  <Story name="With Theme" >
+    <ThemeProvider theme={themeExample}><TenantSelector title="Styled App" onTenantSelected={onTenantSelected} /></ThemeProvider>
```

Comment:
alignment

Reply:
There is a wrapper component. So better showing in the code snippet we need to keep this inline like this.